src/languages/vbnet.js

```diff
@@ -1,63 +1,114 @@
-/*
-Language: Visual Basic .NET
-Description: Visual Basic .NET (VB.NET) is a multi-paradigm, object-oriented programming language, implemented on the .NET Framework.
-Author: Poren Chiang <ren.chiang@gmail.com>
-Website: https://docs.microsoft.com/en-us/dotnet/visual-basic/getting-started/
-*/
+/**
+ * Language: Visual Basic .NET
+ * Description: Visual Basic .NET (VB.NET) is a multi-paradigm, object-oriented programming language, implemented on the .NET Framework.
+ * Authors: Poren Chiang <ren.chiang@gmail.com>, Jan Pilzer
+ * Website: https://docs.microsoft.com/dotnet/visual-basic/getting-started/
+ */
 
 export default function(hljs) {
+  const CHAR = {
+    className: 'string',
+    begin: /"(""|[^/n])"C\b/,
```

Comment:
What is the first case here?

Reply:
Char can either be a single character `"a"C` or an escaped double quote `""""C`.

Reply:
Can we explain with a comment please.